poem/Poem/poem/admin_interface/siteprobes.py

```diff
@@ -238,8 +244,27 @@ def save_model(self, request, obj, form, change):
         if request.user.has_perm('poem.groupown_probe') \
                 or request.user.is_superuser:
             obj.user = request.user.username
-            obj.save()
-            return
+            if form.cleaned_data['new_version'] and change == True or change == False:
```

Comment:
You can leave the clause without check for change={True,False} since it's boolean and you're ok with both values.